incubator/synapse/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+appVersion: 0.0.1
+description: The reference implementation for a Matrix.org chat server
+name: synapse
+version: 0.99.1-rc1
```

Comment:
there is a stable version?

Reply:
They haven't done a 1.0 release of Synapse yet, but also I had the version fields reversed.  I intended for 0.99.1-rc1 to be the appVersion, and the chart version to be 0.0.1

They've also dropped 0.99.1.1 since I originally wrote the chart, so I updated to that while fixing this.